test/client-buildQueryURL.test.ts

```diff
@@ -23,6 +23,8 @@ it("builds a query URL using the master ref", async (ctx) => {
 		ref,
 	})
 	url.searchParams.delete("integrationFieldsRef")
+	url.searchParams.delete("x-c")
+	url.searchParams.delete("x-d")
```

Comment:
❓ #ask: I don't remember why we delete `integrationFieldsRef`. Shouldn't we check for all expected params, including `x-c` and `x-d`?

Reply:
I think it's because each tests gets a random one assigned by the mocked API for `integrationFieldsRef`. I'll rewrite it to expect `x-c` and `x-d`

Reply:
Done!